ruby/conclusion/project_ruby_final.md

```diff
@@ -16,12 +16,14 @@ This is a great project to have as a part of your portfolio going forward becaus
   3. Make it so you can save the board at any time (remember how to serialize?)
   2. Write tests for the important parts.  You don't need to TDD it (unless you want to), but be sure to use RSpec tests for anything that you find yourself typing into the command line repeatedly.
   3. Do your best to keep your classes modular and clean and your methods doing only one thing each.  This is the largest program that you've written, so you'll definitely start to see the benefits of good organization (and testing) when you start running into bugs.
-  4. Have fun!  Check out the [unicode characters](http://en.wikipedia.org/wiki/Chess_symbols_in_Unicode) for a little spice for your gameboard.
-  5. (Optional extension) Build a very simple AI computer player (perhaps who does a random legal move)
+  4. Unfamiliar with Chess? Check out some of the additional resources to help you get your bearings.
```

Comment:
I added this line as I feel it's a good, but vague prompt to check out the Additional Resources section. Here, there will be a like to the previously included "Illustrated rules of Chess" plus the new "Chess notation" link.

The prompt is short, and just gives folks a push to scroll down if they're a bit out of their element.

Reply:
Great idea 💡

Reply:
👍 

I think this is a good first iteration and nothing in the Assignment instructions gives anything away. We can always expand on this if needed.